pkg/ctl/delete/cluster.go

```diff
@@ -28,6 +28,7 @@ func deleteClusterCmd(cmd *cmdutils.Cmd) {
 	cmd.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
 		fs.StringVarP(&cfg.Metadata.Name, "name", "n", "", "EKS cluster name")
 		cmdutils.AddRegionFlag(fs, &cmd.ProviderConfig)
+		cmdutils.AddDisableStackPrefixFlag(fs, &cfg.Metadata.DisableStackPrefix)
```

Comment:
What would happen if a user forgets to give this flag here (or on any delete command)? I am not sure users would expect to need to provide flags/config like this for deletes, does it fail fast enough when it can't find the stack and is then retryable, or could people be left with resources lying around?

Reply:
This seems to tie in with your other comment about only having this flag for creates and automatically discovering clusters/stacks with or without the stack prefix for all other commands. Thinking about it a bit, I really agree with making it an easier experience to use without having to constantly remember about including the cli flag. I'm going to dig in and see if I can figure out a way to handle that better.

If all commands supported using the config file, I would say this is the kind of thing to make config file only and not even include a flag and then any operations on. Do you know if #1126 is going to be completed in the near future? That could be another option to get around this.

Reply:
> Do you know if #1126 is going to be completed in the near future? That could be another option to get around this.

We are a little stretched atm, so this has fallen down our priority list. I feel that making it config only would just improve thie UX a little given the imperative nature of eksctl: people often will throw config together on the fly for a one-off command since each command only processes the conf it cares about.

Annoyingly I had a solution for this issue all mapped out (a little more convoluted, but good UX as you only set the name once) but like a total eejit I wrote it in the community slack and so now the history is gone 😫  🤦‍♀️ . Am trying to scrape it from my memory...